charts/nri-kube-events/templates/deployment.yaml

```
@@ -85,6 +85,10 @@ spec:
                 fieldRef:
                   apiVersion: v1
                   fieldPath: spec.nodeName
+            {{- if .Values.customAttributes }}
+            - name: NRIA_CUSTOM_ATTRIBUTES
+              value: {{ include "newrelic.customAttributes" . | quote }}
+            {{- end }}
```

I think the scope of `"newrelic.customAttributes"` could be reduced by passing just the customAttributes directly:

```suggestion
            {{- with .Values.customAttributes }}
            - name: NRIA_CUSTOM_ATTRIBUTES
              value: {{ include "newrelic.customAttributes" . | quote }}
            {{- end }}
```

WDYT?

---

This was approved on Wednesday and we said that this was going to be released today.

I am sorry to say that I am not going last-minute changes :(